server/opts.go

```diff
@@ -387,6 +389,8 @@ func parseCluster(cm map[string]interface{}, opts *Options) error {
 			opts.Cluster.TLSConfig.ClientAuth = tls.RequireAndVerifyClientCert
 			opts.Cluster.TLSConfig.RootCAs = opts.Cluster.TLSConfig.ClientCAs
 			opts.Cluster.TLSTimeout = tc.Timeout
+		case "cluster_client_advertise":
```

Comment:
Is `cluster_client_advertise` the right name for this? This option should also work for a single standalone server, right? Should it just be `client_advertise`?

Reply:
No, this is really for when a client is notified about a server that joins the cluster, and here is the url to reconnect to. If we are talking standalone server, then client MUST know the url of the server it is connecting to ;-)

Reply:
@kozlovic this is for proxied connections that might not know, and probably shouldn't be leaked internal addresses. I agree with @tylertreat 's suggestion

Reply:
But in standalone, how would you get the info about the server host/ip if you can't connect in the first place!!!